test/court-disputes.js

```diff
@@ -141,9 +141,33 @@ contract('Court: Disputes', ([ poor, rich, governor, juror1, juror2, juror3, arb
       const firstRoundId = 0
 
       beforeEach(async () => {
-        await assertLogs(this.court.createDispute(arbitrable, rulings, jurors, term), NEW_DISPUTE_EVENT)
+        await assertLogs(this.court.createDispute(arbitrable, rulings, jurors, term, { from: poor }), NEW_DISPUTE_EVENT)
       })
 
+      it('gets the correct dispute details', async () => {
+        const dispute = await this.court.getDispute(disputeId)
+
+        assert.strictEqual(dispute[0], arbitrable, 'COURT_INCORRECT_DISPUTE_SUBJECT')
+        assertEqualBN(dispute[1], rulings, 'COURT_INCORRECT_DISPUTE_RULINGS')
+        assertEqualBN(dispute[2], 0, 'COURT_INCORRECT_DISPUTE_STATE')
```

Comment:
Nitpick. A map from the dispute state name to the actual state integer representation can be implemented so states can be referred by name:

```
const DISPUTE_STATES = {
  PRE_DRAFT: 0,
  ADJUDICATING: 1,
  EXECUTED: 2,
  DISMISSED: 3
}
```
This would allow to modify the assertion to:
```suggestion
        assertEqualBN(dispute[2], DISPUTE_STATES.PRE_DRAFT, 'COURT_INCORRECT_DISPUTE_STATE')
```

Reply:
Great idea, will do.